devdoc/docs/ContributeDocs.md

```diff
@@ -0,0 +1,394 @@
+rather than developers.
+
+**Userdoc**/**troubleshoot**: Information related to troubleshooting
+particular components
+
+**userdoc**/**azure**: Useful information about Azure Cloud portal and
+cli
+
+**userdoc**/**roxie**: Useful information for running roxie
+
+**userdoc**/**thor**: Useful information for running thor
+
+**userdoc**/**blogs**: COMING SOON: Location and storage of original
+text for blogs. It also has docs with guidelines and instructions on
+writing Blogs
+
+#### Pull Requests
+
+You can include your documentation with your code in a Pull Request or
+create a separate Jira and Pull Request for the documentation. This
+depends on the size of the code and doc. For a large project, a separate
+Pull request for the documentation is better.
+
+# Writing Style Guide
+
+This section covers the best practice information for writing and
+contributing to the HPCC Systems® Platform documentation.
+
+## General 
+
+We strive to maintain a consistent voice. These guidelines can help your
+writing match that voice.
+
+-   **Use present tense, active voice**. Documentation should be you
+    speaking directly to the reader. Simply tell them what to do.
+
+    This example sentence: "The user selects the file menu." is passive
+    voice. You wouldn’t say it that way in a conversation.
+
+    You should use active voice wording, such as: "Select the file
+    menu".
+
+    Similarly, instructions like these are active voice: "Press the
+    button" or "Submit the file".
+
+    Documentation is you instructing the user. Just tell them what to
+    do.
+
+-   **Be Brief and keep it simple.** Be efficient with your words. Keep
+    sentences short and concise. Keep paragraphs short as well. Use just
+    a few sentences per paragraph. Use simple words wherever possible
+    and try to avoid lengthy explanations.
+
+-   **Consistency:** Be consistent. Use the same voice across all
+    documents. Use the same term, use the same spelling, punctuation,
+    etc. Follow the conventions in this guide.
+
+-   When writing formal documentation that is more than a new feature
+    announcement, do not refer to a new feature, or coming features as
+    such, this does not hold up well over time.
+
+    Do not refer to how things were done in the past. For example, "in
+    the past we had to do X-Y-Z steps and now we no longer have to. This
+    ‘new feature’ can do it in one step, Z". This only adds potential
+    confusion. Just instruct on exactly what needs to be done now, using
+    words efficiently as possible, so for this example just say “perform
+    step Z”
+
+## Terms 
+
+There are many terms specific to HPCC Systems® and the HPCC Systems
+platform. Use the following style guide for word usage and
+capitalization guidelines to use when referring to system components or
+other HPCC-specific entities. Maintain consistent usage throughout all
+docs.
+
+### HPCC Systems® 
+
+Officially and legally the platform's name is HPCC Systems® and it is a
```

Comment:
Is that the name of the platform, or the legal organisation?

Is it "HPCC Systems(R)" or "the HPCC Systems(R) platform"?  The former does not read well - particularly because it uses a plural noun for a singular object.

Reply:
Will clarify. I agree with the plural organization name being awkward in text and conversation.